### PR TITLE
Test public link sharing with longer names

### LINKED ISSUE #32760: create UI tests that check public links with long link names & filenames
we need tests that check if long link names are correctly refused and also some that check if long filenames/foldernames can be used for public links

### PR DESCRIPTION
**Pull request by skshetry**
## Description

This adds a test for public link sharing with longer folder names.

## Related Issue

- Fixes #32760 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Comment by individual-it** [comment]
this PR depends on #32748 that is not merged yet due to CI issues

File: tests/acceptance/features/bootstrap/WebUISharingContext.php [hunk 1]
```diff
@@ -235,6 +235,7 @@ public function theUserCreatesANewPublicLinkForUsingTheWebUIWith(
 
 	/**
 	 * @When the user tries to create a new public link for the file/folder :name using the webUI with
+	 * @When the user tries to create a new public link for the file/folder :name using the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUISharingContext.php]: IMO removing "with" will break an existing test

File: tests/acceptance/features/webUISharingExternal/shareByPublicLink.feature [hunk 2]
```diff
@@ -89,4 +89,19 @@ So that public sharing is limited according to organization policy
 		When the user creates a new public link for the folder "simple-folder" using the webUI with
```
Comment (phil-davis) [inline tests/acceptance/features/webUISharingExternal/shareByPublicLink.feature]: I think this step will now fail - "with" is not in the step definition.

**Review by phil-davis — COMMENTED** [review]
The word "with" at the end of the step text could be optional.

File: tests/acceptance/features/bootstrap/WebUISharingContext.php (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/bootstrap/WebUISharingContext.php]: Sorry, the sstep text is new, and the old step text is also still there!

File: tests/acceptance/features/webUISharingExternal/shareByPublicLink.feature (hunk 2 above)
Comment (phil-davis) [reply tests/acceptance/features/webUISharingExternal/shareByPublicLink.feature]: Not true :)

**Review by phil-davis — APPROVED** [review]

**Comment by skshetry** [comment]
Backport on #32762 .

**Pull request merged**